packages/babel-preset-mc-app/index.js

```diff
@@ -61,6 +61,13 @@ module.exports = function getBabePresetConfigForMcApp() {
           useBuiltIns: true,
         },
       ],
+      [
+        '@emotion/babel-preset-css-prop',
+        {
+          sourceMap: isEnvDevelopment,
```

Comment:
nice 💯

Reply:
I'm wondering though if it makes sense to have a separate babel config for mc "libraries" 🤔

Reply:
Well, to be honest I just ~~stole~~ borrowed these configs from the `ui-kit` 😁